css/index.css

```diff
@@ -1,3 +1,367 @@
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+  margin: 0px 40px 0px 40px;
+}
+@media (max-width: 500px) {
+  html,
+  body {
+    margin: 0;
+  }
+}
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+* {
+  box-sizing: border-box;
+}
+.container {
+  max-width: 800px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+@media (max-width: 800px) {
+  .header {
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+  nav a {
+    justify-content: space-between;
+  }
+}
+@media (max-width: 500px) {
+  .header {
+    flex-direction: column-reverse;
+  }
+}
+nav {
+  display: flex;
+  margin: 20px 0;
+  font-size: 18px;
+}
+nav a {
+  display: flex;
+  color: grey;
+  text-decoration-line: none;
+  margin-left: 20px;
+  justify-content: space-between;
+}
+nav .first-anchor {
+  margin-left: 0;
+}
+@media (max-width: 500px) {
+  nav {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+  nav a {
+    justify-content: center;
+    width: 100%;
+    border-bottom: 1px solid grey;
+    margin-left: 0;
+    margin: 20px, 0;
+  }
+  a {
+    margin: 10px 0;
+    padding-bottom: 10px;
+  }
+}
+.top-content {
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 0;
+}
+@media (max-width: 800px) {
+  .top-content div:nth-of-type(2) {
+    display: none;
+  }
+  .top-content {
+    justify-content: center;
+  }
+}
+@media (max-width: 500px) {
+  .top-content {
+    width: 100%;
+    margin: 0;
+    margin-bottom: 20px;
+  }
+}
+/* all content (except header) left/right margin */
+@media (max-width: 500px) {
+  .top-content,
+  .middle-content,
+  .bottom-content,
+  img.middle-image,
+  .contact,
+  hr {
+    margin: 10px 10px;
+  }
```

Comment:
Be careful adding left and right margins when the width is 100%. This is what is causing horizontal scroll when the site is in mobile (the div is overlapping the page width)